specification/machinelearning/Azure.AI.ChatProtocol/models/blocking.tsp

```diff
@@ -0,0 +1,36 @@
+import "./common.tsp";
+
+namespace Azure.AI.ChatProtocol;
+
+@doc("The representation of a single generated completion.")
+model ChatChoice<TState, TContext> {
```

Comment:
We should constrain TState and TContext or else you risk blowing up when you spread the models.